roles/oVirt.datacenters/README.md

```diff
@@ -3,6 +3,10 @@ oVirt Datacenters
 
 The `oVirt.datacenters` role is used to set up or cleanup oVirt datacenters.
 
+In the special case of cleaning up oVirt datacenter that contains a Hosted Engine VM
+the behavior of the role is different and is explained in more details in:
+[ovirt-datacenter-cleanup]: https://github.com/oVirt/ovirt-ansible/blob/master/roles/ovirt-datacenter-cleanup/README.md
```

Comment:
I think that md file has different syntax for links, see https://github.com/tareqalayan/ovirt-ansible-infra/blob/4867e8a81407a742512abf3276dc07f072f88e95/roles/oVirt.datacenters/README.md
like this most likely?
[Contribution guidelines for this project](https://github.com/oVirt/ovirt-ansible/blob/master/roles/ovirt-datacenter-cleanup/README.md)